src/analyzer/analyzerrhythm.cpp

```diff
@@ -40,7 +40,7 @@ DFConfig makeDetectionFunctionConfig(int stepSize, int windowSize) {
     config.DFType = dfAll;
     config.stepSize = stepSize;
     config.frameLength = windowSize;
-    config.dbRise = 3;
+    config.dbRise = 11;
```

Comment:
The comment above says these values are defaults for the VAMP plugin, so why did you change this? Do you find this gives better results?

Reply:
Yes, manual testing and tuning led to this value. 
It seems to work best with new multi-feature detection.

Reply:
Okay, then add a comment explaining that.